ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,41 @@
+record = (1, 'Grimdiana', 'Bones', 'boulders')
+row = ''
+for x in record:
+  row = row+str(x)+","
+print(row)
+values_list = [1, 1, 2, 3, 5]
+for x in values_list:
+  print(x)
+index_list = []
+for i in range(len(values_list)):
+  index_list.insert(0, i)
+print(index_list)
+for x in index_list:
+  if x%2 != 0:
+    values_list.pop(values_list[x])
```

Comment:
Although the output may look the same, it's actually removing the indexes by their value in `value_list`, not by odd indexes. As a demo of what's going on, try switching it to removing even indexes.

---

![image](https://user-images.githubusercontent.com/16143771/146129297-450f973b-aab2-4bdd-bf35-e8e2a6db1661.png)